\chapter{Introduction}

Genome editing is one of the major technological and biological developments of the past few years and will certainly play a major role in the society of the future. In the past few years the advancements in genome editing have sped up enormously and we are looking for ever more precise tools to keep up with our genome editing needs. One of the newest tools in our collective toolbox is an enzyme named Cas9. It is a CRISPR enzyme that allows targeting of a specific spot on the DNA and cleave the DNA at that exact location. This precision instrument could open the way for a lot of practical uses, for example targeted editing of plant genomes to generate more crops, editing plant genomes to be resistent against pesticides and removing heriditary diseases from embryos. The current issue with Cas9 is that the enzyme is not perfect in its targeting of the DNA, resulting in off-targets which are also cleaved. This could lead to disastrous side-effects if we ever plan to use Cas9 to edit, for example, human embryos.

To solve this issue it is important to understand why some off-targets are cleaved while others are not. A lot of experiments have been done to better understand the off-targeting and people have draw empirical conclusions from the data. However, a physical model would providemore insight and therefore, hopefully, better predictions for the off-targets. The fundamentals for such a physical model have been laid by \cite{Misha}. They are attempting to create a kinetic model which predicts the probability a specific DNA sequence is cleaved by Cas9. This work builds on the fundamentals that have been laid by \cite{Misha}, but will try to answer a slightly different question:

\begin{center}
\emph{"Is it possible to build a more accurate algorithm for predicting the \textbf{binding} of dCas9, based on the physical model from \cite{Misha}, than the existing models based on the empirical observations from experiments, without any underlying physical model?"}
\end{center}

There are several differences between this question and the work done by \cite{Misha}. The main difference is that, while \cite{Misha} predicts the \emph{cleaving} probability of Cas9, this work will attempt to predict the \emph{binding} probability of dCas9. Note the cleavage is done by the enzyme Cas9, while binding is done by dCas9 (dead Cas9), which for all intents and purposes is assumed to be exactly the same enzyme but incapable of cleaving. Even though we are in essence discussing the same process with the same enzyme, \cite{Misha} already notes that 'only binding' and 'binding \& cleaving' are expected to behave differently based on the exact circumstances.

One might ask that if our end goal is genome editing why should we be concerned with the binding probability of dCas9 at all? Would the cleavage probability not be the only parameter of interest? Since Cas9 targets specific DNA sequences in the same way as active Cas9 it has applications outside genome editing. dCas9 can be used for all kinds of research. Usually a fluorescent molecule is attached to the enzyme so it can be tracked and bound (off-)targets on a DNA strand can be identified. This is usefule for sequencing, blocking specific genes, testing DNA strands for specific genes, etc.

In short, the main goal of this work is to answer the question if we can build a binding prediction algorithm  for dCas9 based on \cite{Misha}. While this is the main goal, since it is based on a physical model which requires an understanding of the binding process, implicitly we also hope to get a better understanding of the way (d)Cas9 binds to the DNA.

